src/main/java/uk/nhs/digital/nhsconnect/nhais/service/MeshService.java

```diff
@@ -0,0 +1,73 @@
+package uk.nhs.digital.nhsconnect.nhais.service;
+
+import java.io.IOException;
+
+import javax.jms.JMSException;
+import javax.jms.Message;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import uk.nhs.digital.nhsconnect.nhais.mesh.MeshClient;
+import uk.nhs.digital.nhsconnect.nhais.mesh.MeshConfig;
+import uk.nhs.digital.nhsconnect.nhais.model.mesh.MeshMessage;
+
+import org.apache.qpid.jms.message.JmsBytesMessage;
+import org.apache.qpid.jms.message.JmsTextMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jms.annotation.JmsListener;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@Component @Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class MeshService {
+
+    private final ObjectMapper objectMapper;
+
+    private final MeshClient meshClient;
+
+    private final MeshConfig meshConfig;
+
+//    @JmsListener(destination = "${nhais.amqp.meshOutboundQueueName}")
```

Comment:
This is commented out because it messes up UAT tests (it's taking messages from queue)

Reply:
Maybe just comment out the JmsListener annotation?